platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -200,6 +206,10 @@ private MigrationReportImpl doExport(
       report.record(new MigrationException(Messages.EXPORT_SECURITY_ERROR, exportFile, e));
     } catch (RuntimeException e) {
       report.record(new MigrationException(Messages.EXPORT_INTERNAL_ERROR, exportFile, e));
+    } catch (NoSuchPaddingException e) {
+      report.record(new MigrationException(Messages.EXPORT_INTERNAL_ERROR, keyFile, e));
+    } catch (NoSuchAlgorithmException e) {
```

Comment:
✏️ This branch is identical to the one above it and could be collapsed.

Reply:
Collapsed